phpBB/phpbb/forum/enumeration/forum_options.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\forum\enumeration;
+
+define('FORUM_OPTION_FEED_NEWS', 1);
+define('FORUM_OPTION_FEED_EXCLUDE', 2);
+
+class forum_options
+{
+	/**
+	 * @var int
+	 */
+	const FORUM_OPTION_FEED_NEWS = 1;
```

Comment:
I'm about 90% sure that you can use global constants in class constant declarations on every version of PHP supported by phpBB. It may be a good idea to start specifying the constant's visibility since it's in PHP 7.1.

```php
public const FORUM_OPTION_FEED_NEWS = FORUM_OPTION_FEED_NEWS;
```

Although come to think of it, I think it's a better idea to do it the other way around and define the global constant based on the class constant. It seems more correct and it may make a difference in Opcache's optimizer.

```php
define('FORUM_OPTION_FEED_NEWS', forum_options::FORUM_OPTION_FEED_NEWS);
class forum_options
{
	public const FORUM_OPTION_FEED_NEWS = 1;
}
```

Reply:
the defines were left in by accident.